dali/pipeline/operators/arithmetic/arithmetic.h

```diff
@@ -0,0 +1,268 @@
+// Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef DALI_PIPELINE_OPERATORS_ARITHMETIC_ARITHMETIC_H_
+#define DALI_PIPELINE_OPERATORS_ARITHMETIC_ARITHMETIC_H_
+
+#include <memory>
+#include <string>
+#include <tuple>
+#include <vector>
+
+#include "dali/core/static_switch.h"
+#include "dali/kernels/tensor_shape.h"
+#include "dali/kernels/tensor_shape_print.h"
+#include "dali/kernels/type_tag.h"
+#include "dali/pipeline/operators/arithmetic/arithmetic_meta.h"
+#include "dali/pipeline/operators/arithmetic/expression_impl_factory.h"
+#include "dali/pipeline/operators/operator.h"
+
+namespace dali {
+
+/**
+ * @brief Cover every sample fully.
+ */
+inline std::tuple<std::vector<TileDesc>, std::vector<TileRange>> GetCover(
```

Comment:
suggestion: make a typedef for that tuple type (since it is big enough and you are using it below as well

Reply:
Done